src/content/1.7/development/webservice/_index.md

```diff
@@ -30,7 +30,7 @@ HTTP has several methods that can perform processing on data as defined in the R
 
 ## Enabling & Creating an access to the webservice
 
-Reach the [dedicated page]({{< ref "1.7/development/webservice/tutorials/provide-access" >}}).
+Reach the [dedicated page]({{< ref "1.7/development/webservice/tutorials/1-creating-access" >}}).
```

Comment:
Maybe it's better not to add numbers to the articles. Otherwise if you need to add one in the middle you'll have to rename all the others and create aliases.

Reply:
Yeh that was my big interrogation, the number is a bit more "pedagogic" as it clearly states the order and number of steps to follow
But as you said if we add something in the middle everything goes south.. This refers to another comment above, should I remove all of them? or just on sub sections?

Reply:
Unless I leave them in the title but not in the link 🤔 But as you said there will be a slight difference between the title and H1, but regarding SEO I don't think it's a problem The title often has additional content compared to H1 (site name, product attributes, ...)

Reply:
I think it's best to remove them and use OLs to show numbers if really needed. But it we were to remove them from one place only, remove them from permalinks.